nupic/research/frameworks/vernon/mixins/sync_batchnorm.py

```diff
@@ -0,0 +1,39 @@
+# ----------------------------------------------------------------------
+# Numenta Platform for Intelligent Computing (NuPIC)
+# Copyright (C) 2021, Numenta, Inc.  Unless you have an agreement
+# with Numenta, Inc., for a separate license for this software code, the
+# following terms and conditions apply:
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Affero Public License for more details.
+#
+# You should have received a copy of the GNU Affero Public License
+# along with this program.  If not, see http://www.gnu.org/licenses.
+#
+# http://numenta.org/licenses/
+# ----------------------------------------------------------------------
+import torch.nn as nn
+
+
+class SyncBatchNorm:
+    def create_model(self, config, device):
+        model = super().create_model(config, device)
+        use_synch_batchnorm = config.get("use_synch_batchnorm", True)
```

Comment:
Please add documentation for this mixin, so that it can also easily be later on by others. Also, since you're assuming the config contains certain args like `use_synch_batchnorm`, please add an example config in your docstring like is [here](https://github.com/numenta/nupic.research/blob/master/nupic/research/frameworks/vernon/mixins/dendrite_metrics.py#L97-L107) for this other mixin.

Reply:
I assume 'use_synch_batchnorm' is a typo?

Reply:
Done